src/library/styles/components/_button.scss

```diff
@@ -65,10 +65,6 @@ a {
 
   &.bsds-button-assertive {
     color: $neutral-100;
```

Comment:
this color is also shared by assertive and ghost, so we can condense the whole block for those selectors